src/Campaigns/ServiceProvider.php

```diff
@@ -250,4 +258,19 @@ private function addNewBadgeToMenu(): void
     {
         (new AddNewBadgeToAdminMenuItem())();
     }
+
+    /**
+     * @unreleased
+     */
+    private function registerCampaignCache(): void
+    {
+        give(CacheCampaignData::class)->registerAction();
```

Comment:
Let's remove it and register the hook/action in the service provider.

Reply:
Resolved: https://github.com/impress-org/givewp/pull/8047/commits/368f5aab8e262c224acfa42569fd93e32124cade